esx_service/vmdk_ops_test.py

```diff
@@ -86,6 +86,12 @@ def test_name_parse(self):
             except vmdk_ops.ValidationError as ex:
                 self.assertFalse(expected_result, "Expected vol name parsing to succeed for '{0}'"
                                  .format(full_name))
+    
+def generate_test_vm_name(): 
```

Comment:
nit: better to have method level doc/description that talks about the purpose of this helper